module-ballerina-xsd/Package.md

```diff
@@ -0,0 +1 @@
+## Package Overview
```

Comment:
Lets add these details.

Reply:
I was hoping to send a separate PR for documentation update as this PR already has a lot of file changes